R/add_bloque_marque.R

```diff
@@ -0,0 +1,47 @@
+#' Create a grid graphical object from an image file
+#'
```

Comment:
```suggestion
#'
#' @details Please fill either `file` or `alias` parameter.
```

Reply:
Je te propose de documenter également le fait qu'une erreur survient si les deux paramètres sont renseignés.

```suggestion
#'
#' @details Fill either `file` or `alias` parameter:  an error is thrown if both parameters are provided.
```